frontend/src/APIClients/CrudAPIClient.ts

```diff
@@ -0,0 +1,35 @@
+import baseAPIClient from "./BaseAPIClient";
+import USER_ACCESS_TOKEN_KEY from "../constants/AuthConstants";
+
+const create = async ({ formData }: { formData: any }): Promise<any> => {
```

Comment:
```suggestion
const createEntity = async ({ formData }: { formData: EntityRequest }): Promise<EntityResponse> => {
```
Let's define some types and use them in the function signature. The conversion from `formData` to the request type should happen outside of the API client (in the onSubmit of the component)

```ts
type EntityRequest = {
  stringField: string;
  intField: number;
  ...,
}

type EntityResponse = {
  id: string | number;
  stringField: string;
  intField: number;
  ...,
}
```